pkg/graph/graph.go

```diff
@@ -225,7 +226,7 @@ func (g *Graph) Reverse() *Graph {
 						node.Dependencies = append(node.Dependencies, executor.Name)
 					}
 				}
-				subChartNode.Executors[executor.Name].Executor = subChartNode.Executors[executor.Name].Executor.Reverse()
+				subChartNode.Executors[executor.Name].Executor = subTask.Executors[executor.Name].Executor.Reverse()
```

Comment:
Why is this changed to subtask?

Reply:
Oh shoot I should correct that - I was experimenting